openreview/tools.py

```diff
@@ -703,6 +698,37 @@ def post_submission_groups(client, conference_id, submission_invite, chairs):
             readers=[conference_id, chairs],
             signatories=[]))
 
+def get_submission_invitations(client, open_only=False):
+    '''
+    Returns a list of invitations ids visible to the client caccording to the parameter "open_only".
+    
+    :arg client: Object of :class:`~openreview.Client` class
+    :arg open_only: Default value is False. This is a boolean param with value True implying that the results would be invitations having a future due date.
+    
+    Example Usage:
+    >>> get_submission_invitations(c,True)
+    [u'machineintelligence.cc/MIC/2018/Conference/-/Submission', u'machineintelligence.cc/MIC/2018/Abstract/-/Submission', u'ISMIR.net/2018/WoRMS/-/Submission', u'OpenReview.net/Anonymous_Preprint/-/Submission']
+    '''
+    
+    try:
+        #Calculate the epoch for current timestamp
+        now = int(time.time()*1000)
+        
+        if open_only == True:    
```

Comment:
I would set the minduedate value in the if and do the call get_invitation once so we reduced the repeated code:

```
duedate = now if open_only else None
invitations = client.get_invitations(regex='.*/-/.*[sS]ubmission.*',minduedate=duedate)

```